streams.yml

```diff
@@ -28,6 +28,7 @@ rhel-7-ci-build-root:
   upstream_image: registry.ci.openshift.org/openshift/release:rhel-7-release-openshift-{MAJOR}.{MINOR}
 
 golang:
+  # can be pulled from http://registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder:$TAG
```

Comment:
Yes. Not sure if the explicit path can mess anything.. but that would be ideal

Reply:
Besides triggering another mass rebuild (I think?) I can't see what harm it would cause, but perhaps better to check with @sosiouxme or @jupierce just to be sure.

Reply:
@thegreyd who is the audience for this? Doozer has the logic to do this transformation here: https://github.com/openshift/doozer/blob/b3cc4c32ea06092a74a7c276986472f020e56d24/doozerlib/runtime.py#L1035 .

Reply:
Every other product/component which uses or wants to use golang-builder, or developers who are trying to build containers (with patches) themselves to test a fix etc.  I've lost count of how many times I've seen questions about how to download golang-builder; this would make its proper pullspec self-documenting.  Besides, there is a general trend now anyway to move away from implicit pullspecs.